raiden/tests/unit/transfer/mediated_transfer/test_mediation_fee.py

```diff
@@ -68,27 +68,31 @@ def test_imbalance_penalty():
             (TokenAmount(100), FeeAmount(10)),
         ]
     )
-    assert v_schedule.fee_payer(
+    assert v_schedule.fee_payee(
```

Comment:
Should we add asserts for `fee_payee` here as well?

Reply:
This is `fee_payee`. I've added on case of `fee_payer` to this test that it works as expected. Do you think we should test all values for both? That would be easy enough.

Reply:
I guess it would be nice to have every case for both.